### DISCUSSION TITLE
Error with deployment

### DISCUSSION THREAD

**Discussion by legyul** [alshedivat/al-folio#2141] [Q&A] [answered]
Hi,

I need some help with the deployment.
I followed instructions and when I run "bundle exec jekyll serve --lsi", I got error "did not find expected key while parsing a block mapping at line 5 column 1
/safe_yaml-1.0.5/lib/safe_yaml/load.rb:143:in `parse': (/_config.yml): did not find expected key while parsing a block mapping at line 5 column 1 (Psych::SyntaxError)"
I did not change anything in that line.

How could I fix this?

**Comment by george-gca** [comment, marked as answer]
The error might be misleading you. Checked your `_config.yml` and in [this line](https://github.com/namdarine/namdarine.github.io/blob/96af08fcd7d1f525f204191aae7faa280a8adc39/_config.yml#L10) you should put a `>` sign so it considers what is in the next line, like in our example. If not, it will try to parse this line as a configuration option.

**Reply by legyul** [reply]
Thank you, it got fixed

**Reply by legyul** [reply]
After fixing it, it is not deployed. When I pushed my file, there was no error.

**Reply by george-gca** [reply]
I suggest you follow the install process thoroughly. [Starting from the beginning](https://github.com/alshedivat/al-folio/blob/master/INSTALL.md#installing-and-deploying), making sure that the template is loaded as is before doing any other changes. This way you can more easily pinpoint what changes caused the template to stop working. Also try to do small changes one at a time. If you change a lot of files in a single commit it will be hard to identify the causes.

**Reply by legyul** [reply]
With your suggestion, I followed the instructions, but I got a new error "run fail with prettier.yml"
I appreciate your help.

**Reply by george-gca** [reply]
Check our [FAQ](https://github.com/alshedivat/al-folio/blob/master/FAQ.md#my-code-runs-fine-locally-but-when-i-create-a-commit-and-submit-it-it-fails-with-prettier-code-formatter-workflow-run-failed-for-master-branch-how-do-i-fix-that) for info on this matter.

**Reply by legyul** [reply]
Thank you so much. Everything works fine.
Appreciate your help.
